opossum-tool/OpossumUI
# Remove deprecated constant

### Summary of changes

Delete constant that is not necessary anymore. 

### Context and reason for change

The sources are configured in the input file.

### How can the changes be tested

Open input file in OpossumUI and check that the signal panels have the correct titles.
